docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -567,19 +498,84 @@ Write operations are supported for tables stored on the following systems:
   make sure that no concurrent data modifications are run to avoid data
   corruption.
 
+(delta-lake-schema-table-management)=
+
 ### Schema and table management
 
 The {ref}`sql-schema-table-management` functionality includes support for:
 
-- {doc}`/sql/create-schema`
-- {doc}`/sql/drop-schema`
-- {doc}`/sql/alter-schema`
-- {doc}`/sql/create-table`
+- {doc}`/sql/create-table`, see also {ref}`delta-lake-sql-basic-usage`
 - {doc}`/sql/create-table-as`
 - {doc}`/sql/drop-table`
-- {doc}`/sql/alter-table`
+- {doc}`/sql/alter-table` , see also {ref}`delta-lake-alter-table`
+- {doc}`/sql/create-schema`, see also {ref}`delta-lake-sql-basic-usage`
+- {doc}`/sql/drop-schema`
+- {doc}`/sql/alter-schema`
 - {doc}`/sql/comment`
 
+(delta-lake-sql-basic-usage)=
+
+#### Basic usage examples
```

Comment:
This seems to be all about table and schema management .. move it there .. the title is misleading anyway .. nothing basic about creating schemas and tables..